www/src/public/js/controllers/builder/main.js

```diff
@@ -926,30 +942,37 @@
 
             var importStr = $scope.importModel.input;
             if (importStr) {
-                var listVersion = -1;
+                var listVersion = 3;
 
-                // check if list has a version number
-                var asteriskIdx = listCookieStr.indexOf("*");
-                var tildeIdx = listCookieStr.indexOf("~");
-                if (asteriskIdx < tildeIdx) {
+                // check if list has a version number attached to string.  If so, set listVersion
+                // the -1 check is here get around lists that do not have a list number reporting as a -1 which otherwise causes this function to always run and remove the first letter of lists that do not have a version number
```

Comment:
We don't need to comment every submission we do, that's what commit messages are for. Typically, you'll only want to comment code if it's a bit of code that will be really confusing without it, or if you want to put a reference link to where you found some information that will probably be needed when reading the code in the future.